_getting-started/concepts-system-overview-guide.md

```diff
@@ -0,0 +1,394 @@
+
+      This doesn't mean that Stitch doesn't perform transformations during the replication process. Stitch just doesn't perform _arbitrary_ transformations - Stitch will perform only the transformations required to ensure the loaded data is useful and compatible with your destination. The transformations Stitch performs are dependent on the destination you choose. These can include:
+
+      {% for subsection in section.subsections %}
+      - [{{ subsection.title }}](#{{ subsection.anchor }})
+      {% endfor %}
+
+      Stitch's philosophy is that what you do with your data depends on your needs, and by keeping data close to its original form, Stitch enables you to manage and transform it as you see fit. While we don't support user-defined transformations inside of Stitch, you can take advantage of [Talend's transformation and data quality solutions](https://www.stitchdata.com/platform/datatransformation/){:target="new"} to design and integrate your own transformations.
+
+    subsections:
+      - title: "Data typing"
+        anchor: "data-typing"
+        example-data:
+          - id: "1"
+            boolean: "true"
+            string: ""
+          - id: "2"
+            boolean: ""
+            string: "yes"
+          - id: "3"
+            boolean: ""
+            string: "no"
+        content: |
+          Stitch's data typing process consists of three steps which take place during replication:
+
+          1. **Extraction**: Identify the data type in the source
+          2. **Preparing**: Map the data type to a Stitch data type
+          3. **Loading**: Convert the Stitch data type to a destination-compatible data type
+
+          Stitch converts data types only where needed to ensure the data is accepted by your destination.
+
+          With some exceptions, when a data type is changed in the source, Stitch will create an additional column in the destination to accommodate the new data type. This will look like the column has been "split". For example:
+
+          {% assign column-headings = "id|has_magic__bo|has_magic__st" | split: "|" %}
+          {% assign attributes = "id|boolean|string" | split: "|" %}
+
+          <table class="attribute-list">
+          <tr>
+          {% for column-heading in column-headings %}
+          <td>
+          <strong>{{ column-heading }}</strong>
+          </td>
+          {% endfor %}
+          </tr>
+          {% for example in subsection.example-data %}
+          <tr>
+          {% for attribute in attributes %}
+          <td>
+          {{ example[attribute] }}
+          </td>
+          {% endfor %}
+          </tr>
+          {% endfor %}
+          </table>
+
+          Stitch handles changed data types in this way to ensure previously loaded data is retained in its original format. We recommend using views to coerce data types when this occurs.
```

Comment:
I think the view recommendation is good in some cases, but not all, and maybe not even the majority of cases

Reply:
Keep it or toss it?

Maybe something like "Some Stitch customers use views to coerce data types when this occurs." if we vote to keep it.

Reply:
I'm supportive of keeping it with the modification you suggested